### DISCUSSION TITLE
🔗 Dynamic grid for posts (Duplicate of #25256)

### DISCUSSION THREAD

**Discussion by realrecordzLab** [elementor/elementor#27423] [Feature Request]
### Prerequisites

- [ ] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I'm using elementor from long time and it's a great way to build fast wordpress websites. I like it and it's very complete, but sometimes it will become difficult to realize things that designers will create. For a website where there is a portfolio page, I had the problem to re-create the design that was given to me from a team member of my company.

### Describe the solution you'd like.

<img width="971" alt="lN3UlN9F" src="https://github.com/elementor/elementor/assets/51746386/05b11fc5-0db8-43b3-8fff-507555765890">

In the container I've used as you can see from the image, I have the columns that are be placed in a particular way. What will be nice to have is to be able to have the ability to use this layout or the others to show custom post or posts.

### Describe alternatives you've considered.

At the moment the only way to use that layout is to write php shortcode and html/CSS to have the design. I've also tried using multiple posts widget with custom query but seems not working as exected.

### Agreement

- [x] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hello there, @realrecordzLab!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #25256

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards
